src/lib.rs

```diff
@@ -33,11 +33,12 @@
     external_doc
 )]
 
-#[cfg(target_arch = "aarch64")]
+//#[cfg(target_arch = "aarch64")]
```

Comment:
we'll need those you can test with: `TARGET=aarch64-unknown-linux-gnu cargo +nightly test` to get it to use the right target